Sources/DNA/Color/Color.swift

```diff
@@ -61,7 +61,7 @@ public extension UIColor {
         return UIColor(r: 104, g: 226, b: 184)!
     }
 
-    private convenience init?(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) {
+    internal convenience init?(r: CGFloat, g: CGFloat, b: CGFloat, a: CGFloat = 1.0) {
```

Comment:
Internal is the default, so should be fine to remove it.

Reply:
Is it really when the extension has explicit public access control? 🤔
Since Amir said earlier that he didn't intend that the extension should be made explicitly public I'll remove that and and the internal for the initialiser 😄